src/app/(privateProfile)/profitMoney/page.tsx

```diff
@@ -1,8 +1,104 @@
+"use client";
+
+import { Key } from "react";
+import { useForm } from "react-hook-form";
+
+import { PlusIcon } from "../../../assets/script/expenses/PlusIcon";
+
+import Input from "../../../ui/input/Input";
+import { InputType } from "../../../helpers/Input";
+import { IExpensesInputForm } from "../../../types/pages/Expenses";
+import { Select } from "../../../ui/select/Select";
+import Button from "../../../ui/button/button";
+
+import IncomeTransaction from "../../../components/userProfileLayout/incomeTranstaction/incomeTransaction";
+import { incomeTransactions } from "../../../mocks/IncomeTransaction";
+
+import { formatMoney } from "../../../utils/formatMoney";
+
+import style from "./profitMoney.module.scss";
+
 function ProfitMoney() {
+	const { control } = useForm<IExpensesInputForm | any>({
+		defaultValues: {
+			sum: "",
+		},
+		mode: "all",
+		delayError: 200,
+	});
+
+	const incomeMoney = 200000;
 	return (
-		<>
-			<div>ProfitMoney</div>
-		</>
+		<div className={style.pageWrap}>
+			<div className={style.pageContainer}>
+				<form className={style.formContentWrapper}>
+					<div className={style.container}>
+						<h1 className={style.headerTitle}>Доходы</h1>
+						<div className={style.byDateContainer}>
+							<div className={style.totalMonthlyWrapper}>
+								<p className={style.totalMonthlyWrapper__month}>Общий доход за Январь</p>
+								<p className={style.totalMonthlyWrapper__sum}>{formatMoney(incomeMoney, 0)}</p>
+							</div>
+							<div className={style.dateSelectionWrapper}>
+								<p className={style.dateSelectionWrapper__description}>Выбор даты</p>
+								<Input control={control} label={"Выбор даты"} type={InputType.Date} name={"date"} />
```

Comment:
can you replace Input component with AppInput component here and in other similar places of your changes?